velox/vector/tests/VectorTestBase.h

```diff
@@ -166,6 +166,22 @@ class VectorTestBase {
     return vectorMaker_.flatVector<T>(size, type);
   }
 
+  FlatVectorPtr<ShortDecimal> makeDecimalFlatVectorNullable(
```

Comment:
inconsistent naming: makeNullableFlatVector vs. makeDecimalFlatVectorNullable

perhaps, rename to makeNullableFlatDecimalVector